### DISCUSSION TITLE
[Suggestion] Make Diving Knives Attachable to Firearms as Bayonets

### DISCUSSION THREAD

**Discussion by Arctodites** [FakeFishGames/Barotrauma#13227] [Ideas and suggestions]
I think it would be cool if diving knives could be affixed to firearms such as the rifle or the assault rifle like bayonets, allowing the guns to be used as melee weapons.  Knives could even share the same slot already used by flashlights.

It would be a bonus if this meant doing-away with the large, red 'out of ammo' warning that pops up every time you brandish an empty firearm -- because let's be honest: sometimes you just want to feign that you have a loaded weapon, or sometimes you just want to use an empty weapon for its attached flashlight.

**Comment by SomeRandomNoobKekeke** [comment]
and which button to stab then?
